### DISCUSSION TITLE
Contribute code

### DISCUSSION THREAD

**Discussion by lukechen116** [sqids/sqids-java#2] [Show and tell]
Hello @4kimov, I implemented Java code for Sqids.  I would like to contribute this code and upload branch. How do I provide?

**Comment by fanweixiao** [comment]
Hi @lukechen116, just pull request

**Comment by 4kimov** [comment]
@lukechen116 Thanks for reaching out. Followed up [here](https://github.com/sqids/sqids-java/issues/1).
